databuilder/clients/neptune_client.py

```diff
@@ -0,0 +1,141 @@
+# Copyright Contributors to the Amundsen project.
```

Comment:
Let me know if you all want this to be placed in: https://github.com/amundsen-io/amundsengremlin